src/app/examples/buttons/buttons.html

```diff
@@ -34,6 +34,51 @@
     </button>
   </div>
 
+  <div class="d-flex flex-wrap align-items-center gap-4 mb-6">
+    <div class="btn-split" role="group" aria-label="Split button">
+      <button type="button" class="btn btn-primary text-truncate" [disabled]="disabled">
+        <i class="icon element-download"></i>
```

Comment:
should be `aria-hidden` everywhere for icons so that it doesn't show up in accessibility tree snapshot